docs/services/slam/cartographer/_index.md

```diff
@@ -541,20 +387,13 @@ This example JSON configuration:
     {{% /tab %}}
     {{< /tabs >}}
 
-    For more information about the configuration attributes, see [Attributes](#attributes).
+    For more information about the configuration attributes, see [Attributes](#attributes). If you want to configure cartographer to use a locally saved map, see [Using locally built maps](#using-locally-built-maps).
 
-1.  Start a mapping session:
+2. After configuring cartographer on the machine, the map should appear automatically. Navigate to the **CONTROL** tab on your machine's page and click on the dropdown menu matching the `name` of the service you created.
 
-    Navigate to the **CONTROL** tab on your machine's page and click on the dropdown menu matching the `name` of the service you created.
+   ![slam RC card localize only](/services/slam/slam-RC-card-localize-only.png)
```

Comment:
[optional] Can remove the image, we don't show an image for "create new map" or "update existing"